.golangci.yaml

```diff
@@ -0,0 +1,13 @@
+# https://golangci-lint.run/usage/configuration/#config-file
+
+run:
+  allow-parallel-runners: true
+  timeout: 30s
+
+linters:
+  enable:
+    - gocritic
+    - gocyclo
+    - gofmt
+    - gosec
+    - govet
```

Comment:
also curious if there's a rationale behind just these out of all the others at https://golangci-lint.run/usage/linters/

as in, addresses a few fat concerns and that's about it?

Reply:
> as in, addresses a few fat concerns and that's about it?

That's precisely the reason. We might add others as we go